src/lib/Scenes/Artwork/Components/CommercialButtons/BidButton.tsx

```diff
@@ -16,26 +16,11 @@ export interface BidButtonProps {
 const registrationWasAttempted = sale => !!sale.registrationStatus
 const isRegisteredToBid = sale => registrationWasAttempted(sale) && sale.registrationStatus.qualifiedForBidding
 const watchOnly = sale => sale.isRegistrationClosed && !isRegisteredToBid(sale)
+const getMyLotStanding = artwork => artwork.myLotStanding && artwork.myLotStanding[0]
```

Comment:
Minor style thing, but if you make these class methods you can access props directly and avoid having to pass the objects in each time. (That _feels_ clearer to me but I'm not an expert)

Reply:
I believe this could also error if myLotStanding is an empty array. May want to check do this: `artwork.myLotStanding && artwork.myLotStanding.length && artwork.myLotStanding[0]`

Reply:
@sweir27 i kind of thought not using props makes it little more obvious what this method gets and how those attributes are defined, but if you feel strong i can move them to class.